CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
I've found a way to find out whenever CMAKE_INSTALL_PREFIX was set by the user -- CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT. But available only in CMake 3.6+.

I will go with SNAPPY_INSTALL_ON_WINDOWS.